### PR TITLE
Add test results: 20220517_233826

### PR DESCRIPTION
**Pull request by srv-gh-o11y-gdi**
Add test results: 20220517_233826

### CODE REVIEW

File: results/20220517_233826/results.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+Startup time:
+  No Instrumentation: "NaN seconds"
```
Comment (mateuszrzeszutek) [inline results/20220517_233826/results.yaml:3]: Looks like something has failed horribly... @breedx-splk do you have any idea what might be the cause of it?

Comment (breedx-splk) [reply results/20220517_233826/results.yaml:3]: Yup, looks awful. Will investigate.

Comment (breedx-splk) [reply results/20220517_233826/results.yaml:3]: Oh, I see it now. This is a symptom of the agent names [being hard coded in the yaml persister](https://github.com/signalfx/splunk-otel-java-overhead-test/blob/main/src/test/java/io/opentelemetry/results/YamlSummaryPersister.java#L52). I will see if there's a reasonable way to consistently generate those more dynamically.

**Comment by breedx-splk** [comment]
Going to try and fix the yaml generator

**Comment by breedx-splk** [comment]
Nevermind, since this is a one-off, I think we should just merge it. We will end up reverting these configs, and the next run will have a proper yaml file.

**Review by breedx-splk — APPROVED** [review]

**Pull request closed without merging**